src/components/user/SignUpForm/index.js

```diff
@@ -37,6 +42,16 @@ const SignUpForm = ({ handleSubmit, error, navigator }) => {
         component={Input}
         password
       />
+      <Field
+        name="gender"
+        label={translate('SIGN_UP.gender')}
+        component={PickerScreen}
```

Comment:
It's the PickerScreen actually a full screen? If it is, then this is OK. But if it isn't, then you can rename it to... `CustomPicker` or similar?

Reply:
I'll rename it because it is like a normal field